### ISSUE TITLE
Parsing error after reinstalling eslint v4.10.0 probably caused by acorn 5.2.0 dependecy

### ISSUE THREAD

**Issue by doberkofler** [eslint/eslint#9536]
**Tell us about your environment**

* **ESLint Version:** 4.10.0
* **Node Version:** 8.8.1
* **npm Version:** 4.6.1
* **acorn Version:** 5.2.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

{
    "parserOptions": {
        "ecmaVersion": 2017
    },
	"env": {
        "es6": true
    },
    "extends": "eslint:recommended",
    "globals": {
    },
    "plugins": [
    ],
    "rules": {
        // Possible Errors
		"for-direction": "error",
		"no-await-in-loop": "error",
        "no-console": "off",
        "no-debugger": "warn",
        "no-extra-boolean-cast": "warn",
		"no-extra-parens": "off",
		"no-unsafe-finally": "error",
		"no-prototype-builtins": "off",
		"no-template-curly-in-string": "error",
        "valid-jsdoc": ["warn", {
			"prefer": {
				"return": "returns",
				"argument": "param",
				"arg": "param",
				"constructor": "class",
				"virtual": "abstract"
			},
			"preferType": {
				"Boolean": "boolean",
				"Number": "number",
				"String": "string",
				"object": "Object",
				"array": "Array",
				"date": "Date"
			},
			"matchDescription": ".+",
            "requireParamDescription": true,
            "requireReturnDescription": true,
            "requireReturn": false,
			"requireReturnType": true
        }],

        // Best Practices
		"array-callback-return": "warn",
		"class-methods-use-this": "warn",
		"complexity": ["warn", 30],
        "consistent-return": "warn",
        "curly": ["error", "all"],
        "default-case": "error",
        "dot-notation": ["error", {"allowKeywords": true, "allowPattern": "^[a-zA-Z]+(_[a-zA-Z]+)+$"}],
        "eqeqeq": "error",
        "guard-for-in": "error",
        "no-alert": "off",
        "no-caller": "error",
        "no-case-declarations": "error",
        "no-div-regex": "error",
        "no-else-return": "warn",
        "no-empty-pattern": "warn",
        "no-eq-null": "error",
        "no-eval": "error",
        "no-extend-native": "error",
        "no-extra-bind": "error",
		"no-extra-label": "warn",
        "no-fallthrough": "warn",
        "no-floating-decimal": "error",
        "no-implicit-coercion": ["warn", {"boolean": true, "number": true, "string": true}],
		"no-implicit-globals": "warn",
        "no-implied-eval": "error",
        "no-iterator": "error",
        "no-labels": ["error", {"allowLoop": false, "allowSwitch": false}],
        "no-lone-blocks": "warn",
        "no-loop-func": "error",
        "no-magic-numbers": "off",
		"no-multi-spaces": ["warn", {"ignoreEOLComments": true}],
        "no-multi-str": "error",
        "no-native-reassign": "error",
        "no-new": "error",
        "no-new-func": "error",
        "no-new-wrappers": "error",
        "no-octal-escape": "error",
        "no-octal": "error",
        "no-process-env": "error",
        "no-proto": "error",
        "no-redeclare": "error",
		"no-restricted-globals": [
			"error",
			// event is globally available but causes problems when used (e.preventDefault()) in a jQuery callback
			// but has not been defined as a parameter to the callback
			"event",
			"jQuery"
		],
		"no-restricted-properties": [
			"error", {
				"object": "$",
				"property": "ajax",
				"message": "Use framework/backend/ajax."
	        },
			{
				"object": "$",
				"property": "each",
				"message": "Use native es2015 forEach."
	        },
			{
				"object": "$",
				"property": "camelCase"
	        },
			{
				"object": "$",
				"property": "cookie"
	        },
			{
				"object": "$",
				"property": "error"
	        },
			{
				"object": "$",
				"property": "extend"
	        },
			{
				"object": "$",
				"property": "get",
				"message": "Use framework/backend/ajax."
	        },
			{
				"object": "$",
				"property": "getJSON",
				"message": "Use framework/backend/ajax."
	        },
			{
				"object": "$",
				"property": "getScript",
				"message": "Use framework/backend/ajax."
	        },
			{
				"object": "$",
				"property": "globalEval"
	        },
			{
				"object": "$",
				"property": "grep"
	        },
			{
				"object": "$",
				"property": "inArray"
	        },
			{
				"object": "$",
				"property": "isArray"
	        },
			{
				"object": "$",
				"property": "isEmptyObject"
	        },
			{
				"object": "$",
				"property": "isFunction"
	        },
			{
				"object": "$",
				"property": "isNumeric"
	        },
			{
				"object": "$",
				"property": "isPlainObject"
	        },
			{
				"object": "$",
				"property": "makeArray"
	        },
			{
				"object": "$",
				"property": "map",
				"message": "Use native es2015 map."
	        },
			{
				"object": "$",
				"property": "merge"
	        },
			{
				"object": "$",
				"property": "noop"
	        },
			{
				"object": "$",
				"property": "now"
	        },
			{
				"object": "$",
				"property": "parseJSON"
	        },
			{
				"object": "$",
				"property": "parseXML"
	        },
			{
				"object": "$",
				"property": "post",
				"message": "Use framework/backend/ajax."
	        },
			{
				"object": "$",
				"property": "trim"
	        },
			{
				"object": "$",
				"property": "type"
	        },
			{
				"object": "$",
				"property": "proxy",
				"message": "Use native Function.bind."
	        },
			{
				"object": "$",
				"property": "when"
	        },
			{
				"object": "$",
				"property": "jGrowl"
	        },
			{
				"object": "$",
				"property": "buttonset",
				"message": "Use $.controlgroup"
			},
			{
				"object": "$",
				"property": "size"
			},
			{
				"object": "$",
				"property": "unique",
				"message": "Use $.uniqueSort"
			},
			{
				"object": "$",
				"property": "bind",
				"message": "Use $.on"
			},
			{
				"object": "$",
				"property": "unbind",
				"message": "Use $.off"
			},
			{
				"object": "$",
				"property": "delegete",
				"message": "Use $.on"
			},
			{
				"object": "$",
				"property": "undelegate",
				"message": "Use $.off"
	        }
		],
        "no-return-assign": "error",
        "no-script-url": "error",
		"no-self-assign": "error",
        "no-self-compare": "error",
        "no-sequences": "error",
        "no-throw-literal": "error",
		"no-unmodified-loop-condition": "warn",
        "no-unused-expressions": "error",
		"no-unused-labels": "error",
        "no-useless-call": "error",
        "no-useless-concat": "warn",
		"no-useless-return": "warn",
        "no-void": "error",
        "no-warning-comments": "off",
        "no-with": "error",
        "radix": "error",
        "vars-on-top": "error",
        "wrap-iife": ["error", "outside"],
        "yoda": ["error", "never", {"exceptRange": true}],

        // Strict Mode
        "strict": ["error", "global"],

        // Variables
        "no-catch-shadow": "error",
        "no-delete-var": "error",
        "no-label-var": "error",
        "no-shadow-restricted-names": "error",
        "no-shadow": ["warn", {"builtinGlobals": false}],
        "no-undef": "error",
        "no-undef-init": "error",
        "no-undefined": "warn",
        "no-unused-vars": ["warn", {"vars": "all", "args": "after-used"}],
        "no-use-before-define": ["error", "nofunc"],

        // Node.js and CommonJS
        "global-require": "warn",
        "handle-callback-err": "warn",
		"no-buffer-constructor": "warn",
        "no-new-require": "warn",
        "no-path-concat": "warn",

        // Stylistic Issues
		"array-bracket-newline": "off",
        "array-bracket-spacing": ["warn", "never"],
        "block-spacing": ["warn", "never"],
        "brace-style": ["warn", "1tbs"],
        "camelcase": "warn",
        "comma-spacing": ["warn", {"before": false, "after": true}],
        "comma-style": ["warn", "last"],
        "computed-property-spacing": ["warn", "never"],
        "consistent-this": ["warn", "that"],
        "eol-last": "warn",
		"func-call-spacing": ["warn", "never"],
        "func-names": "off",
        "func-style": "off",
		"function-paren-newline": ["warn", "multiline"],
		"id-blacklist": "off",
        "id-length": "off",
        "id-match": "off",
        "indent": ["warn", "tab", {"SwitchCase": 1}],
        "jsx-quotes": "off",
        "key-spacing": ["warn", {"beforeColon": false, "afterColon": true}],
		"keyword-spacing": ["error", {"before": true, "after": true, "overrides": {}}],
        "linebreak-style": "off",
        "lines-around-comment": "off",
        "max-depth": ["warn", 10],
        "max-len":  ["warn", 500],
		"max-lines": ["warn", 1000],
        "max-nested-callbacks": ["warn", 10],
        "max-params": ["warn", 10],
        "max-statements": ["warn", 150],
		"max-statements-per-line": ["warn", {"max": 2}],
        "new-cap": "warn",
        "new-parens": "warn",
		"newline-per-chained-call": "off",
        "no-array-constructor": "warn",
		"no-bitwise": "warn",
        "no-continue": "warn",
        "no-inline-comments": "off",
        "no-lonely-if": "off",
        "no-mixed-spaces-and-tabs": "warn",
        "no-multiple-empty-lines": "off",
		"no-multi-assign": "warn",
        "no-negated-condition": "off",
        "no-nested-ternary": "warn",
        "no-new-object": "warn",
		"no-plusplus": "off",
        "no-restricted-syntax": ["warn", "WithStatement"],
        "no-ternary": "off",
        "no-trailing-spaces": "warn",
        "no-underscore-dangle": "off",
        "no-unneeded-ternary": "warn",
		"no-whitespace-before-property": "warn",
		"object-curly-newline": "off",
        "object-curly-spacing": ["warn", "never"],
		"one-var-declaration-per-line": "off",
        "operator-assignment": "off",
        "operator-linebreak": ["warn", "after", {"overrides": {"?": "ignore", ":": "ignore"}}],
        "padded-blocks": "off",
		"padding-line-between-statements": ["warn",
			// require blank lines before all return statements
			//{"blankLine": "always", "prev": "*", "next": "return"},
			//  require blank lines after every sequence of variable declarations
//			{"blankLine": "always", "prev": ["const", "let", "var"], "next": "*"},
//			{"blankLine": "any",    "prev": ["const", "let", "var"], "next": ["const", "let", "var"]},
			//  switch/case
//			{"blankLine": "always", "prev": ["*"], "next": "case"},
//			{"blankLine": "any",    "prev": ["switch"], "next": ["case"]},
//			{"blankLine": "always", "prev": ["*"], "next": "default"},
//			{"blankLine": "any",    "prev": ["switch"], "next": ["default"]},
			// require blank lines after all directive prologues
			{"blankLine": "always", "prev": "directive", "next": "*"},
			{"blankLine": "any",    "prev": "directive", "next": "directive"}
		],
        "quote-props": ["warn", "consistent"],
        "quotes": ["warn", "single", "avoid-escape"],
        "require-jsdoc": "off",
        "semi": ["warn", "always"],
        "semi-spacing": ["warn", {"before": false, "after": true}],
		"semi-style": ["error", "last"],
		"sort-imports": "off",
        "sort-vars": "off",
        "space-before-blocks": "warn",
        "space-before-function-paren": ["warn", {"anonymous": "always", "named": "never"}],
        "spaced-comment": ["off", "always"],
		"switch-colon-spacing": ["error", {"after": true, "before": false}],
		"unicode-bom": ["error", "never"],
        "wrap-regex": "off",

        // ECMAScript 6
        "arrow-body-style": ["warn", "as-needed"],
        "arrow-parens": ["warn", "always"],
        "arrow-spacing": ["warn", {"before": true, "after": true}],
        "constructor-super": "error",
        "generator-star-spacing": ["warn", {"before": true, "after": false}],
        "no-class-assign": "error",
        "no-confusing-arrow": "error",
        "no-const-assign": "error",
        "no-dupe-class-members": "error",
		"no-duplicate-imports":  "error",
        "no-new-symbol": "error",
        "no-this-before-super": "error",
        "no-useless-constructor": "error",
		"no-useless-computed-key": "warn",
		"no-useless-rename": "warn",
        "no-var": "error",
        "object-shorthand": "off",
        "prefer-arrow-callback": "off",
		"prefer-const": "warn",
        "prefer-reflect": "off",
        "prefer-rest-params": "off",
        "prefer-spread": "off",
        "prefer-template": "off",
        "require-yield": "error",
        "template-curly-spacing": "error",
        "yield-star-spacing": "error"
    }
}


```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js


// @flow

/*::
import type {UiForm$ConfigType} from '../UiForm';
*/

import * as ODS from '../../../ods/js/ods';
import * as UiElement from '../../uielement/UiElement';
import {loadConfig} from './persist';

/*::
import type {UiForm$ConfigPageType} from '../UiForm';

export type UiForm$InternalConfigType = {
	config: {
		id: string,
		focus: string,
		alwaysRender: boolean,
		title: string,
		class: string,
		description: string,
		method: string,
		action: string,
		feedback: boolean,
		classes: string,
		wrapperClasses: string,
		addErrorContainersToEachElement: boolean,
		pages: Array<UiForm$ConfigPageType>
	},
	formHiddenObject: {[string]: UiElement.Hidden},
	formElementObject: {[string]: UiElement.Element},
	memorizedValues: {[string]: string},
	buttonsArray: Array<any>
};
*/

const compareFormAtLeast = {
		id: 'string',
		pages: []
	},
	compareFormAtMost = {
		id: 'string',
		alwaysRender: 'boolean',
		focus: 'string',
		title: 'string',
		method: 'string',
		description: 'string',
		action: 'string',
		feedback: 'boolean',
		classes: 'string',
		wrapperClasses: 'string',
		addErrorContainersToEachElement: 'boolean',
		buttons: [],
		hidden: [{
			id: 'string',
			type: 'string',
			value: 'arbitrary'
		}],
		pages: []
	};

const comparePageAtLeast = {
		rows: []
	},
	comparePageAtMost = {
		title: 'string',
		class: 'string',
		type: 'string',
		open: 'boolean',
		hidden: 'boolean',
		rows: []
	};

const compareRowAtLeast = {
		elements: []
	},
	compareRowAtMost = {
		label: 'arbitrary',
		subLabel: 'string',
		class: 'string',
		hidden: 'boolean',
		elements: []
	};

const compareButtonAtLeast = {
		buttonText: 'string',
		buttonClass: 'string'
	},
	compareButtonAtMost = {
		buttonText: 'string',
		buttonClass: 'string',
		buttonListener: 'function',
		buttonIcon: 'string',
		buttonType: 'string'
	};

/**
*	Validate the given configuration object, add default values and return a cloned version
*
*	@param {Object} config - Configuration object.
*	@returns {UiForm$ConfigType} - Internal configuration object.
*/
export function validate(config/*: UiForm$ConfigType*/)/*: UiForm$InternalConfigType*/ {
	if (arguments.length !== 1 || !_.isObject(config)) {
		throw new TypeError('Parameter "config" must be an object');
	}

	const NEW_CONFIG = {
		config: {
			id: '',
			focus: '',
			alwaysRender: false,
			title: '',
			class: '',
			description: '',
			method: 'post',
			action: '',
			feedback: false,
			classes: '',
			wrapperClasses: '',
			addErrorContainersToEachElement: false,
			pages: []
		},
		formHiddenObject: {},
		formElementObject: {},
		memorizedValues: {},
		buttonsArray: []
	};

	// Validate "form" configuration
	ODS.Types.checkParameterDeep(config, 'config', compareFormAtLeast, 'atLeast');
	ODS.Types.checkParameterDeep(config, 'config', compareFormAtMost, 'atMost');

	// Validate "buttons" configuration
	if (typeof config.buttons !== 'undefined') {
		ODS.Types.checkParameterDeep(config.buttons, 'config.buttons', [compareButtonAtLeast], 'atLeast');
		ODS.Types.checkParameterDeep(config.buttons, 'config.buttons', [compareButtonAtMost], 'atMost');
	}

	// Validate the "id" of the form
	ODS.Types.checkIsNonemptyString(config.id, 'config.id');

	// Initialize the object properties
	NEW_CONFIG.config.id = config.id;
	if (typeof config.focus !== 'undefined') {
		NEW_CONFIG.config.focus = config.focus;
	}
	if (typeof config.alwaysRender !== 'undefined') {
		NEW_CONFIG.config.alwaysRender = config.alwaysRender;
	}
	if (typeof config.title !== 'undefined') {
		NEW_CONFIG.config.title = config.title;
	}
	if (typeof config.class !== 'undefined') {
		NEW_CONFIG.config.class = config.class;
	}
	if (typeof config.description !== 'undefined') {
		NEW_CONFIG.config.description = config.description;
	}
	if (typeof config.method !== 'undefined') {
		NEW_CONFIG.config.method = config.method;
	}
	if (typeof config.action !== 'undefined') {
		NEW_CONFIG.config.action = config.action;
	}
	if (typeof config.feedback !== 'undefined') {
		NEW_CONFIG.config.feedback = config.feedback;
	}
	if (typeof config.classes !== 'undefined') {
		NEW_CONFIG.config.classes = config.classes;
	}
	if (typeof config.wrapperClasses !== 'undefined') {
		NEW_CONFIG.config.wrapperClasses = config.wrapperClasses;
	}
	if (typeof config.addErrorContainersToEachElement !== 'undefined') {
		NEW_CONFIG.config.addErrorContainersToEachElement = config.addErrorContainersToEachElement;
	}

	if (typeof config.buttons !== 'undefined') {
		NEW_CONFIG.buttonsArray = config.buttons;
	}

	// validate all id's
	validateIDs(config);

	// Validate pages
	validatePages(config, NEW_CONFIG);

	// Process the hidden elements
	NEW_CONFIG.formHiddenObject = validateHidden(config);

	ODS.DLG_Form.setFormSubmit(NEW_CONFIG.config.id, true);

	return NEW_CONFIG;
}

/*
*	Validate the pages, rows and elements
*/
function validatePages(config/*: Object*/, newConfig) {
	// Load open pages
	const openPages = loadConfig(config.id, config.pages.length);

	// Process "pages"
	config.pages.forEach(function (pageConfig, pageIndex) {
		const PAGE_NAME = 'page#' + pageIndex;

		// Validate "page"
		ODS.Types.checkParameterDeep(pageConfig, PAGE_NAME, comparePageAtLeast, 'atLeast');
		ODS.Types.checkParameterDeep(pageConfig, PAGE_NAME, comparePageAtMost, 'atMost');

		const PAGE = {
			title: pageConfig.title || '',
			class: pageConfig.class || '',
			type: '',
			open: true,
			hidden: pageConfig.hidden || false,
			rows: []
		};

		// Validate "type"
		if (typeof pageConfig.type !== 'undefined') {
			if (pageConfig.type === 'accordion') {
				PAGE.type = pageConfig.type;
			} else {
				throw new TypeError('Invalid type "' + pageConfig.type + '" in ' + PAGE_NAME);
			}
		}

		// Validate "open"
		if (PAGE.type === 'accordion') {
			// get the default from form definition
			if (typeof pageConfig.open !== 'undefined') {
				if (typeof pageConfig.open === 'boolean') {
					PAGE.open = pageConfig.open;
				} else {
					throw new TypeError('Invalid open "' + pageConfig.open + '" in ' + PAGE_NAME);
				}
			}

			// get what we stored in the cookie
			if (openPages !== null) {
				PAGE.open = openPages[pageIndex];
			}
		}

		newConfig.config.pages.push(PAGE);

		// Process "rows"
		pageConfig.rows.forEach(function (rowConfig, rowIndex) {
			const ROW_NAME = PAGE_NAME + ' row# ' + rowIndex;

			// Validate "row"
			ODS.Types.checkParameterDeep(rowConfig, ROW_NAME, compareRowAtLeast, 'atLeast');
			ODS.Types.checkParameterDeep(rowConfig, ROW_NAME, compareRowAtMost, 'atMost');

			const ROW = {
				label: (typeof rowConfig.label === 'undefined') ? null : rowConfig.label,
				subLabel: rowConfig.subLabel || '',
				class: rowConfig.class || '',
				hidden: rowConfig.hidden || false,
				elements: []
			};
			PAGE.rows.push(ROW);

			// Process "elements"
			rowConfig.elements.forEach(function (elementConfig) {
				const ELEMENT = new UiElement.Element(elementConfig);

				newConfig.formElementObject[ELEMENT.getId()] = ELEMENT;
				ROW.elements.push(_.clone(elementConfig));
			});
		});
	});
}

/*
*	Validate the hidden fields
*/
function validateHidden(config/*: Object*/)/*: Object*/ {
	const HIDDEN = {};

	if (typeof config.hidden !== 'undefined') {
		config.hidden.forEach(function (element) {
			const ELEMENT = new UiElement.Hidden(element);
			HIDDEN[ELEMENT.getId()] = ELEMENT;
		});
	}

	return HIDDEN;
}

/*
*	Validate ID's
*/
function validateIDs(config/*: Object*/) {
	const IDS = [];

	// "regular" elements
	if (_.isArray(config.pages)) {
		config.pages.forEach(function (pageConfig, pageIndex) {
			if (_.isArray(pageConfig.rows)) {
				pageConfig.rows.forEach(function (rowConfig, rowIndex) {
					if (_.isArray(rowConfig.elements)) {
						rowConfig.elements.forEach(function (elementConfig, elementIndex) {
							if (IDS.indexOf(elementConfig.id) === -1) {
								IDS.push(elementConfig.id);
							} else {
								throw new TypeError(`Duplicate id "${elementConfig.id}" found in page# ${pageIndex}, row# ${rowIndex}, element# ${elementIndex}`);
							}
						});
					}
				});
			}
		});
	}

	// "hidden" elements
	if (_.isArray(config.hidden)) {
		config.hidden.forEach(function (elementConfig, elementIndex) {
			if (IDS.indexOf(elementConfig.id) === -1) {
				IDS.push(elementConfig.id);
			} else {
				throw new TypeError(`Duplicate id "${elementConfig.id}" found in hidden element# ${elementIndex}`);
			}
		});
	}
}


```

```bash

D:\MyDev\ljs_app\trunk\periscope\build>node_modules\.bin\eslint src\framework\uiform\form\config.js

D:\MyDev\ljs_app\trunk\periscope\build\src\framework\uiform\form\config.js
  0:0  error  Parsing error: Cannot read property 'override' of undefined

✖ 1 problem (1 error, 0 warnings)

```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**

Before reinstalling eslint the code (and actually) any other source code was linted as expected without any error or warning.

After reinstalling eslint the error `Parsing error: Cannot read property 'override' of undefined` started to appear in 100% of the files.

After examining the problems, I discovered that the `acorn` dependency was upgraded from 5.1.1 to 5.2.0. Reverting acorn back to 5.1.1 confirmed by assumption and it there seems to be a problem or incompatibility with the acorn 5.2.0 version that was just released some 6 hours ago.

**Comment by doberkofler** [comment]
The problem has been confirmed as a problem in acorn 5.2.0 and was wondering if eslint should not depend on the exact acorn version to prevent this kind of problem in its key dependency.

**Comment by adrianheine** [comment]
acorn 5.2.1 fixes the issue. The dependency on acorn is through espree.

**Comment by platinumazure** [comment]
We could conceivably release a new espree version that will depend on ^5.2.1 to fix this for ESLint users. Thanks for the issue!

**Comment by doberkofler** [comment]
acorn 5.2.1 fixed the parser error but unfortunately eslint now reports an out of memory condition as follows:

```cmd
<--- Last few GCs --->

[36028:00000000002F5C10]   157081 ms: Mark-sweep 1504.3 (1821.9) -> 1504.3 (1821.9) MB, 115.8 / 0.0 ms  allocation failure scavenge might not succeed
[36028:00000000002F5C10]   157223 ms: Mark-sweep 1504.3 (1821.9) -> 1504.3 (1802.9) MB, 141.4 / 0.0 ms  last resort GC in old space requested
[36028:00000000002F5C10]   157357 ms: Mark-sweep 1504.3 (1802.9) -> 1504.3 (1802.4) MB, 133.3 / 0.0 ms  last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 000001AAAF825E91 <JSObject>
    2: replace(this=000001525116C919 <Very long string[14696]>,000001525116C941 <JSRegExp <String[5]: [^\t]>>,000001AAAF86ECE9 <String[1]:  >)
    4: /* anonymous */(aka /* anonymous */) [D:\MyDev\ljs_app\trunk\periscope\build\node_modules\babel-code-frame\lib\index.js:42] [bytecode=000003EF060FA671 of
fset=119](this=00000398E0002311 <undefined>,line=0000015251174F91 <Very long string[33248]>,i...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
```

I'm currently investigation if this is also caused by the acorn update.

**Comment by doberkofler** [comment]
The "out of memory condition" was a false alert that seems to have been caused by a lot of uninstalling and installing of npm packages. Something got messed up but after reinstalling all dependencies from scratch the "out of memory condition" is gone as well.

**Comment by Aladdin-ADD** [comment]
thank you very much! @doberkofler

we have upgrade `acorn@5.2.1` in `espree`, just need to release a new version, and upgrade `espree`.
refs: #9546

**Comment by ilyavolodin** [comment]
Closing this issue, since we released v4.11.0 with the new espree that fixes this problem.
